administrative-sdk/connection.js

```diff
@@ -192,32 +193,34 @@ class Connection {
    * @param {callback} [ecb] The callback that handles the error response.
    * @param {string} [auth] The authorization header value to pass along with the request.
    */
-  _ajaxGet(url, cb, ecb, auth) {
-    var self = this;
+  _ajaxGet(url, auth) {
     var request = new XMLHttpRequest();
```

Comment:
Part of converting to promises is to rewrite XMLHttpRequest to the new [fetch](https://github.com/github/fetch) API.